trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class TrimmerDecorator < Nameable
+  def initialize(nameable = nameable.new)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- [ ] Here slice only when greater then a certain length, Kindly use this
`@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`